cmd/templ/lspcmd/proxy/server.go

```diff
@@ -228,6 +234,61 @@ func (p *Server) Initialize(ctx context.Context, params *lsp.InitializeParams) (
 		Save:              &lsp.SaveOptions{IncludeText: true},
 	}
 
+	for _, c := range params.WorkspaceFolders {
+		path := strings.TrimPrefix(c.URI, "file://")
+		werr := filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			uri := uri.URI("file://" + path)
+			isTemplFile, goURI := convertTemplToGoURI(uri)
+			if isTemplFile {
```

Comment:
Sorry for not spotting this last time, it's a small nit, but I think inverting this if will improve readability and reduce indentation.

Reply:
Agreed, done.